doc/source/dev/testing.rst

```diff
@@ -3,18 +3,17 @@
 Testing
 =======
 
-Continuous integration with Travis-CI
-------------------------------------
+Continuous integration with GitHub Actions
+------------------------------------------
 
-The project is using `Travis-CI <https://travis-ci.org/PyWavelets/pywt>`_ service
-for continuous integration and testing.
+The project is using GitHub Actions for continuous integration and testing.
 
 Current build status is:
 
 .. image::
-    https://secure.travis-ci.org/PyWavelets/pywt.png?branch=master
+    https://github.com/PyWavelets/pywt/actions/workflows/tests/badge.svg?branch=master
```

Comment:
This link is broken, it should be https://github.com/PyWavelets/pywt/actions/workflows/tests.yml/badge.svg?branch=master I believe.

Reply:
Fixed.